clientconn.go

```diff
@@ -924,22 +919,24 @@ func (ac *addrConn) connect() error {
 	return nil
 }
 
+// equalAddress returns true is a and b are considered equal.
+// This is different from the Equal method on the resolver.Address type which
+// considers all fields to determine equality. Here, we only consider fields
+// that are meaningful to the subConn.
+func equalAddress(a, b resolver.Address) bool {
```

Comment:
These should probably be pointers to avoid the copy.

Reply:
Done.